### PR TITLE
[Expert] Astral Shuffle

### PR DESCRIPTION
**Pull request by MuteTiefling**
Goal is to reshuffle astral a bit to make some of the tools available earlier, while moving the most powerful stuff later without breaking progression. This does mean that anything other than Bright constellations are unavailable for quite some time, as well as personal attunement. But this way the various rods and crystal lenses can be used earlier.  I'll spend some time reviewing other items to see what can be shuffled around with these preliminary changes.

Move Starlight Crafting Altar forward:
image

Move Altar of Birthing to Luminous Crafting Table
image

Runic Altar too: 
image

Move Celestial Altar behind Elven Portal
image

Attunement Altar becomes a sacrifice ritual
image

### CODE REVIEW

**Comment by MuteTiefling** [comment]
Ender Hopper: 
image

**Comment by MuteTiefling** [comment]
Bit of blood magic/Astral Crossover for constellation specific shifting stars:
image

generic shifting star: 
image

lenses:
Regen
image
Damage
image
Ignition
image
Break
image
Growth
image
Push

**Comment by MuteTiefling** [comment]
Mana spreaders moved to Luminous Crafting
image

Elven Spreader
image

Gaia Spreader
image

Mana Collector
image


Pushing back prior Apoth shelf recipes as the altar changes made them available earlier than desired
image
image
image

Add Komodo dragons to Desert and Savanna biomes as they appear to be very rare when spawning in badlands only.

**Comment by MuteTiefling** [comment]
More lightning for buuz. Increases prismarine to firmament ratio 
image

**Comment by MuteTiefling** [comment]
Ritual Pedestal: 
image



Tossing some more dark utils expertification in here too
image

Vector Plates
image

Fast Vector Plates
image

Extreme Vector Plates
image

Import Plates upgrade from vector plates:
image

Disable Export plates because frankly this just seems dangerous for TPS >.> Better to use any other item transport

Moved all the 'rune' type effect plates into Alchemy. Each uses a suitable potion and a sigil (sigil not consumed)
image

**Comment by MuteTiefling** [comment]
Remove Dust > Mana Powder and Dye > Mana Powder. Only obtained via crushed source gems

File: config/alexsmobs/komodo_dragon_spawns.json [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
+                "type": "BIOME_DICT",
+                "negate": false,
+                "value": "overworld"
+            },
+            {
+                "type": "BIOME_CATEGORY",
+                "negate": false,
+                "value": "desert"
+            }
+        ]
```
Comment (NillerMedDild) [inline config/alexsmobs/komodo_dragon_spawns.json:38]: You can merge these into the same array and save a little size :)

**Review by NillerMedDild — APPROVED** [review]

File: config/alexsmobs/komodo_dragon_spawns.json (hunk 1 above)
Comment (MuteTiefling) [reply config/alexsmobs/komodo_dragon_spawns.json:38]: Ah cool. Wasn't too sure so I just copied from one of the others

**Pull request merged**